src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -249,11 +249,15 @@ public Diagnostic DisallowedProperty(bool warnInsteadOfError, Symbol? sourceDecl
                     ? $" from source declaration \"{sourceDeclaration.Name}\""
                     : string.Empty;
 
+                var typeInaccuracyClause = isResourceSyntax
+                    ? $" If this is an inaccuracy in the documentation, please report it to the Bicep Team: https://aka.ms/bicep-type-issues"
```

Comment:
Terminate this string with a "."

Reply:
done + moved link from plain text into a clickable option.